src/Spice86.Core/Emulator/OperatingSystem/DosPathResolver.cs

```diff
@@ -0,0 +1,235 @@
+namespace Spice86.Core.Emulator.OperatingSystem;
+
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Spice86.Core.Emulator.OperatingSystem.Structures;
+using Spice86.Shared.Interfaces;
+using Spice86.Shared.Utils;
+
+/// <inheritdoc />
+public class DosPathResolver : IDosPathResolver {
+
+    /// <inheritdoc />
+    public IDictionary<char, MountedFolder> DriveMap { get; private set; } = new Dictionary<char, MountedFolder>();
+
+    /// <inheritdoc />
+    public char CurrentDrive { get; private set; }
+
+    /// <inheritdoc />
+    public string CurrentHostDirectory {
+        get => DriveMap[CurrentDrive].CurrentFolder;
```

Comment:
On windows this returns a path with backslashes while other functions return paths with forward slashes.

Reply:
Fixed.